docs/performances.md

```diff
@@ -0,0 +1,169 @@
+# Performances analysis
```

Comment:
This doc is really great!
One small remark is that you seem to really dislike ending dots at the end of sentences :laughing:

Reply:
Fixed [here](https://github.com/cozy/cozy-client/compare/13bb965aca7d1ced675c7e4fa5bc7b0fc6e3bd4c..ee1b801325c7d7db3ca12d34712541c1b9b95165)